### ISSUE TITLE
[Accessibility] VoiceOver reads nothing for items under Pocket Recommendations or Most Visited sites

### ISSUE THREAD

**Issue by mozillamobilebot** [mozilla-mobile/firefox-ios#4990]
From https://bugzilla.mozilla.org/show_bug.cgi?id=1453909

STR:
1. Run Firefox 11 for iOS.
2. Open a new tab so the Home screen gets displayed.
3. Turn on VoiceOver.
4. Touch the heading "Pocket recommendations".
5. Swipe to the right. First there's the button to show More.
6. Swipe to the right again through the items.

Expected: VoiceOver should read each item in a comprehensive manner.

Actual: VoiceOver reads nothing. The item gets highlighted on the screen, but nothing is read.

Same is true for the Most Visited sites down below.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-2111)

**Comment by SimonBasca** [comment]
For me this is only partially reproducible after following the STR:

1. Run Firefox 11 for iOS.
2. Open a new tab so the Home screen gets displayed.
3. Turn on VoiceOver.
4. Touch the heading "Pocket recommendations".
5. Swipe to the right. First there's the button to show More.
6. Swipe to the right again through the items.

Actual results:
- VoiceOver doesn't read the title when when first tapping on a pocket entry and also some pocket titles are only partially read by VoiceOver or just the bottom text is read "trending".

**Comment by data-sync-user** [comment]
➤ Simion Basca commented:

Marking as verified fix on main b83ce4fd11fcee4508a9dd83dd2c68473ca9c096. Pocket items are reading correctly with Voice Over enabled.
